src/cow-react/utils/number.ts

```diff
@@ -0,0 +1,18 @@
+import { DECIMAL_SEPARATOR } from '@cow/constants/format'
+
+export function decomposeDecimal(decimal: string, exactDecimals?: number): [string, string] {
```

Comment:
It would be great to add a unit test for this with few examples of input/output values.